Library/Formula/rust.rb

```diff
@@ -23,6 +42,34 @@ def install
     system "./configure", *args
     system "make"
     system "make install"
+
+    resource("cargo").stage do
+      cargo_stage_path = pwd
```

Comment:
In such a case it could be extremely useful for:
- `FileUtils.mktemp` to yield the path it just created
- `Resource.unpack` (and thus `Resource.stage`) to yield not just `self` but also the path yielded by `mktemp`.